android-base/src/main/java/io/github/droidkaigi/confsched2020/di/RepositoryComponentModule.kt

```diff
@@ -55,23 +55,24 @@ object RepositoryComponentModule {
         context: Context,
         droidKaigiApi: DroidKaigiApi,
         googleFormApi: GoogleFormApi,
-        database: SessionDatabase,
+        sessionDatabase: SessionDatabase,
         sponsorDatabase: SponsorDatabase,
         announcementDatabase: AnnouncementDatabase,
         staffDatabase: StaffDatabase,
         contributorDatabase: ContributorDatabase,
         firestore: Firestore
     ): RepositoryComponent {
-        return RepositoryComponent.builder()
-            .context(context)
-            .droidKaigiApi(droidKaigiApi)
-            .googleFormApi(googleFormApi)
-            .database(database)
-            .sponsorDatabase(sponsorDatabase)
-            .firestore(firestore)
-            .announcementDatabase(announcementDatabase)
-            .staffDatabase(staffDatabase)
-            .contributorDatabase(contributorDatabase)
-            .build()
+        return RepositoryComponent.factory()
+            .create(
+                context,
```

Comment:
Thank you for awesome PR!
Can we use named argument for this?👀

Reply:
Of course, I'll change it!

Reply:
done with 5ab0afa